py3-build.yaml

```diff
@@ -44,3 +44,22 @@ update:
     identifier: pypa/build
     use-tag: true
     strip-suffix: .post1
+
+test:
+  environment:
+    contents:
+      packages:
+        - busybox
+  pipeline:
+    - runs: |
+        LIBRARY="build"
+        IMPORT_STATEMENT="import build"
+
+        if ! python -c "$IMPORT_STATEMENT"; then
+            echo "Failed to import library '$LIBRARY'."
+            python -c "$IMPORT_STATEMENT" 2>&1
+            exit 1
+        else
+            echo "Library '$LIBRARY' is installed and can be imported successfully."
+            exit 0
+        fi
```

Comment:
Can we factor this out to be similar to "git-checkout" named pipeline?

something like
```yaml
- uses: py-import
   imports: [....] # list of imports implied ${{package.name}}, without the py3 prefix?
```

Cause Ideally, we would want to factor out the common code like this. Such that we can improve and extend it everywhere at once. Similarly for the steps to `build` python package too.

Reply:
but i see this is mentioned already in the pull request description. Never mind.